### PR TITLE
Add .coverage to generated .gitignore

### LINKED ISSUE #2: Add .coverage to .gitignore
It would be nice to have .coverage added to .gitignore by default.

### PR DESCRIPTION
**Pull request by spladug**
Fixes #2 

:eyeglasses: @pacejackson

### CODE REVIEW

File: baseplate_cookiecutter/{{cookiecutter.project_slug}}/.gitignore [hunk 1]
```diff
@@ -2,6 +2,8 @@
 *.pyo
 __pycache__
 
+.coverage/
```
Comment (pacejackson) [inline baseplate_cookiecutter/{{cookiecutter.project_slug}}/.gitignore]: I think this is a file, not a folder.

**Comment by pacejackson** [comment]
💅  Just one small thing, thanks for adding this!

File: baseplate_cookiecutter/{{cookiecutter.project_slug}}/.gitignore (hunk 1 above)
Comment (spladug) [inline baseplate_cookiecutter/{{cookiecutter.project_slug}}/.gitignore]: woops, you're totally right. i dunno where i got that idea.

**Comment by spladug** [comment]
:haircut:

**Comment by pacejackson** [comment]
🐟

**Pull request merged**